src/Microsoft.OData.Core/ODataLibraryCompatibility.cs

```diff
@@ -4,29 +4,39 @@
 // </copyright>
 //---------------------------------------------------------------------
 
-using System.Diagnostics.CodeAnalysis;
+using System;
 
 namespace Microsoft.OData
 {
     /// <summary>
     /// Library compatibility levels.
     /// </summary>
-    [SuppressMessage("Microsoft.Design", "CA1008:EnumsShouldHaveZeroValue")]
-    public enum ODataLibraryCompatibility
+    [Flags]
+    public enum ODataLibraryCompatibility : long
     {
+        /// <summary>
+        /// No specific version or feature set.
+        /// </summary>
+        None = 0, // No flags set
+
+        /// <summary>
+        /// When enabled, the CSDL Scale and SRID "variable" value will be written "Variable" instead of "variable" for compatibility with older versions of the library.
+        /// </summary>
+        UseLegacyVariableCasing = 1 << 0,
+
         /// <summary>
         /// Version 6.x
         /// </summary>
-        Version6 = 060000,
+        Version6 = 1 << 4,
 
         /// <summary>
         /// Version 7.x
         /// </summary>
-        Version7 = 070000,
+        Version7 = 1 << 5,
 
         /// <summary>
         /// The latest version of the library.
         /// </summary>
-        Latest = int.MaxValue
+        Latest = 1 << 6,

```

Comment:
I don't think you should change this

Reply:
Any reason for this? Why can't it be used like other flags?

Reply:
My reasoning is that `Latest` should mean you don't want to apply any legacy behavior. I don't think we should bitwise this enum value with other flags. To me, `Latest` is like a sentinel value. I did come across some references where the sentinel value is assigned `int.MaxValue`